Makefile

```
@@ -8,6 +8,12 @@ RUN ?= .*
 PKG ?= ./...
 
 
+.PHONY: lint-pipeline
+lint-pipeline:
+	black .drone.star
```

running `black` alone writes out the changes.
```suggestion
	black --diff --check .drone.star
```

---

Yeah, this was an operator convenience since it is the build file. I suppose the pipeline could bail out if the pipeline was updated and not linted and formatted but that seems like the domain of a precommit hook.